lifecycle_test.go

```diff
@@ -446,6 +449,19 @@ func sendGpgKeyPair() {
 	coreutils.ExitOnErr(errorutils.CheckResponseStatusWithBody(resp, body, http.StatusCreated))
 }
 
+func importRb(t *testing.T, rbName, rbVersion string) {
+	wd, err := os.Getwd()
+	assert.NoError(t, err)
+	exportedFileName := rbName + "-" + rbVersion + ".zip"
+	exportedFilePath := path.Join(wd, rbName, rbVersion, exportedFileName)
+	output := lcCli.RunCliCmdWithOutput(t, "rbi", exportedFilePath)
+	var importResp string
+	if !assert.NoError(t, json.Unmarshal([]byte(output), &importResp)) {
+		return
+	}
+	assert.Equal(t, "", importResp)
```

Comment:
Checking the output is empty is a bit pointless and doesn't indicate success. 
Rather focus on verifying the release bundle is actually in Artifactory. You can do it using `getReleaseBundleSpecification` for example (assert non existing before import, existing after)